Added date & time to requests in Sample Test

khavinshankar opened the pull request:
Fixes #1829 

- [x] added date_of_sample and date_of_result to Sample Test card
- [x] setting date_of_result to new Date() when the result is entered in Sample Test
- [x] setting date_of_sample to new Date() when the Sample Test request is created

**Note:  _The date and time of Test and Result will be shown on cards that are created from now on because all previous cards have null values for date_of_sample and date_of_result._**

![image](https://user-images.githubusercontent.com/29787772/134310464-514718f4-c8be-4fe3-8aa0-ec281ee2c460.png)

abhinandanarya06 reviewed (commented):
I think `created_by_user` and `modified_by_user` field is missing according to the issue. 
Verify if above fields are necessary or given only just for example

Other Looks good to me

nihal467 commented:
![lskdnnksd](https://user-images.githubusercontent.com/57055998/134800345-28e9f807-9f76-4bf2-889d-664c97dcf544.PNG)

I only submitted a request for a sample test, but the time & date mentioned in the "tested on" is the sample requested time, is it right?

khavinshankar commented:
> @khavinshankar I only submitted a request for a sample test, but the time & date mentioned in the "tested on" is the sample requested time, is it right?

sorry that was a mistake,

- [x] changed Tested On to Date of Sample
- [x] changed Results On to Date of Result
- [x] fixed the styling (made it small, bold and grey)
- [x] also shows "Not Available" where the dates are missing

![image](https://user-images.githubusercontent.com/29787772/135107356-66cf4378-7e6e-4117-84a4-18af74ad5e99.png)

samholics commented:
tested, verified

now able to see request submitted date&time as date_of_sample also date_of_result when update test result from labs end.
